lib/Unicode/UCD.pm

```diff
@@ -4180,7 +4180,9 @@ RETRY:
             } # End of loop constructing the converted list
 
             # Finish up the data structure for our converted swash
-            my $type = ($second_try eq 'nfkccf') ? 'ToNFKCCF' : 'ToDm';
+            my $type = ($second_try =~ /\b ( nfkcs?cf ) \b /x)
```

Comment:
Why `\b` and not full begin/end anchors?

Reply:
sloppiness